docs/license.rst

```diff
@@ -13,20 +13,38 @@
 Using and Citing SORA
 =====================
 
-Terms of Use
-------------
+Citing
+------
 
 
 SORA is free software and we kindly ask users that make use of it (or part of it), especially in 
```

Comment:
Consider rewriting the paragraph. 
"SORA is free software, and we kindly ask users that make use of it (or of part of it), especially in projects that result in scientific publications, to include the following citation:"